spinnaker-gradle-project/build.gradle

```diff
@@ -51,6 +51,11 @@ tasks.register('clean') {
 }
 
 tasks.register('publish') {
-  dependsOn ":spinnaker-project-plugin:publish"
-  dependsOn ":spinnaker-extensions:publish"
+  dependsOn ":spinnaker-project-plugin:publishPlugins"
+  dependsOn ":spinnaker-extensions:publishPlugins"
+}
+
+// Alias for publish
```

Comment:
Why do we need this if the github action invokes the `publish` task?

Reply:
We don't - I had that in before deciding changing the workflow was cleaner.  Removed it.